src/core/qgsxmlutils.cpp

```diff
@@ -78,6 +116,39 @@ QDomElement QgsXmlUtils::writeMapUnits( Qgis::DistanceUnit units, QDomDocument &
   return unitsNode;
 }
 
+QDomElement QgsXmlUtils::writeBox3D( const QgsBox3D &box, QDomDocument &doc, const QString &elementName )
+{
```

Comment:
This is quite a verbose way of storing the box in xml -- let's use a single XML element with xmin/xmax / etc stored as attributes instead of child elements.

Reply:
I agree, I did not know if attribute or sub nodes are preferred (as for the writeRectangle one)

Reply:
done